lib/metasploit/framework/credential_collection.rb

```diff
@@ -113,6 +113,7 @@ def each_unfiltered
           pass_from_file.chomp!
           yield Metasploit::Framework::Credential.new(private: pass_from_file, realm: realm, private_type: private_type(pass_from_file))
         end
+        pass_fd.seek(0)
```

Comment:
Can we update the tests to include this scenario 👀

Reply:
yup for sure